src/stdlib.js

```diff
@@ -196,6 +202,86 @@ defunTyped('not', ['boolean'], ['boolean'], function (stack, scope) {
 
 // Non-standard functions
 
+/* This function clones an object. Not sure if it belongs here, I will probably change this later.
+See: http://stackoverflow.com/questions/728360/most-elegant-way-to-clone-a-javascript-object */
+
+function clone(obj) {
```

Comment:
Yeah, I know it's overkill. It's temporary anyway, I just wanted it to work :)

Hmm, I tried to just push the same object twice when setting, but editing it's properties would also edit the properties of all the other objects.